src/components/dialog-manager-test.js

```diff
@@ -129,6 +129,18 @@ describes.realWin('DialogManager', {}, (env) => {
     expect(graypaneStubs.destroy).to.not.be.called;
   });
 
+  it('should catch view error', async () => {
+    const view = {
+      whenComplete: () =>
+        Promise.reject(new DOMException('cancel', 'AbortError')),
+    };
+    try {
+      await dialogManager.handleView(view);
+    } catch (error) {
+      expect(error).to.equal('cancel');
```

Comment:
I don't think this test is working as expected, since this line isn't covered. Could you try using this pattern?

```js
await expect(linkSaveFlow.getRequestPromise()).to.be.rejectedWith(
  /Both authCode and token are available/
);```

Reply:
I changed the code to return the statement and it made it easier to test.

Reply:
Interesting, I'm not seeing the code change

Reply:
badabing